panels/logbook/ha-panel-logbook.html

```diff
@@ -73,6 +79,7 @@
           disabled='[[isLoading]]'
           required
         ></vaadin-date-picker>
+        <paper-icon-button id='refresh-logbook' icon='mdi:refresh' on-click='refreshLogbook' hidden$='[[_isRefreshHidden(isLoading, _currentDate)]]'></paper-icon-button>
```

Comment:
ID seems not to be used.

Reply:
Yes, classes are always preferred. Because IDs have to be unique, you will have to change the HTML if you want to reuse the style for another element.